client/src/components/profile/index.jsx

```diff
@@ -141,6 +167,12 @@ class Profile extends Component {
     this.setState({ activeIndex: newIndex });
   }
 
+  /**
+   * Parse data and calls the updateProfile action creator
+   *
+   * @memberof Profile
+   * @returns {null} Nothing
```

Comment:
why is the JSDOCs return variable Nothing?

Reply:
It is returning nothing because it is a just function that calls a redux action creator